<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>595</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
border:none;
}
QWidget{
	
	
	background-color: rgb(218, 231, 164);
	
}

QPushButton:hover
{
	background-color: rgb(114, 218, 211);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(194, 206, 146);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">icon: url(:/Resource/openDir.png);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">icon: url(:/Resource/playlist_order.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">icon: url(:/Resource/prevSong.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">icon: url(:/Resource/pause.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">icon: url(:/Resource/nextSong.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">icon: url(:/Resource/volume.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 199, 141);</string>
   </property>
  </widget>
  <widget class="QSlider" name="playCourseSlide">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="curLable">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="totalLable">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
